lib/views/home/home_view.dart

```diff
@@ -113,28 +114,29 @@ class _HomeViewState extends State<HomeView> {
   }
 
   Future<List<dynamic>?> listPayments() async {
-    final invoicesList = await widget.provider.get<AppApi>().listInvoices();
     final fundsList = await widget.provider.get<AppApi>().listFunds();
+    final listPays = await widget.provider.get<AppApi>().listTransaction();
+    List<dynamic> list = [];
 
-    var listPayments = List.from(invoicesList.invoice)..addAll(fundsList!.fund);
+    /// Adding all the funds and transactions we are getting from the json
+    list.addAll(fundsList!.fund);
+    list.addAll(listPays.transactions);
+    LogManager.getInstance
+        .debug("In home view : ${fundsList.fund} ${listPays.transactions}");
 
     /// FIXME: sort the payments list
-    return listPayments;
+    return list;
   }
 
   Widget _buildSpecificPaymentView(
       {required BuildContext context,
       required List<dynamic> items,
       required int index}) {
-    return items[index].identifier == "invoice"
+    return items[index].identifier == "transaction"
         ? Column(
             crossAxisAlignment: CrossAxisAlignment.start,
             children: [
-              Text("Amount: ${items[index].amount}"),
-              Text("Confirmed: ${items[index].status}"),
-              Text("Bolt11: ${items[index].bolt11}"),
-              Text("Payment Hash: ${items[index].paymentHash}"),
-              Text("Paid time: ${items[index].paidTime}"),
+              Text("Transaction Hash : ${items[index].txId}"),
```

Comment:
No this is true for the bitcoin transaction but not for the lightning payment, if you send a payment to someone or receive a payment you had other information

to get you the payment you need to run the following command `lightning-cli --testnet listpays status=complete` (outgoing payment, the payment that you made

or to get the payment that you receive you need to do `lightning-cli --testnet listinvoices` and get the payment the status completed

and then you need to should also the output (that are the bitcoin transaction) from `listfunds`

Reply:
I was using this command `lightning-cli listtransactions --testnet`. Oh, this command shows the payments made within a channel. Okay I'll update the code